homeassistant/components/roku/media_player.py

```diff
@@ -40,27 +39,24 @@
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
-    """Set up the Roku platform."""
-    if not discovery_info:
-        return
-
-    host = discovery_info[CONF_HOST]
-    async_add_entities([RokuDevice(host)], True)
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Roku config entry."""
+    roku = hass.data[DOMAIN][entry.entry_id][DATA_CLIENT]
+    async_add_entities([RokuDevice(roku, entry[CONF_HOST])], True)
 
 
 class RokuDevice(MediaPlayerDevice):
     """Representation of a Roku device on the network."""
 
-    def __init__(self, host):
+    def __init__(self, roku, ip_address, enabled_default=True):
```

Comment:
where do you ever pass `enabled_default`=False ?

Reply:
I think I was just being forward thinking but I'll remove since not directly using